ocrd-olena-binarize

```diff
@@ -375,7 +375,7 @@ function main {
     # shellcheck source=../core/ocrd/bashlib/lib.bash
     source $(ocrd bashlib filename)
     ocrd__wrap "$SHAREDIR/ocrd-tool.json" "ocrd-olena-binarize" "$@"
-    ocrd__minversion 2.55.2
+    ocrd__minversion 2.58
```

Comment:
I meant this should probably be 2.58.0

Reply:
Well spotted, kudos. Yes, I fixed that.

Reply:
I run a test and it failed because of this with lib.bash from core, but I was not 100% sure that this was the cause.